libfreerdp/codec/h264.h

```diff
@@ -60,6 +60,7 @@ extern "C"
 		UINT32 FrameRate;
 		UINT32 QP;
 		UINT32 NumberOfThreads;
+		UINT32 VaapiEnabled; /* 0 means disabled, != 0 means enabled */
```

Comment:
Rename to generic hardware acceleration reqeusted option. might be a good idea to leave this open for future extensions to request specific accelerators, so define this as `0 -> off`, `1 -> generic on` and other values for future use

Reply:
Renamed.